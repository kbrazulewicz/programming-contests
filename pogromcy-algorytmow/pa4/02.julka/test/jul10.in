1000000000000000000000000000000000000000000000000000000000000000000000020000002000000000002020222222
9999998
