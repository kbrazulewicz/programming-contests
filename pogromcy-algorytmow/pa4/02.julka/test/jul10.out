500000000000000000000000000000000000000000000000000000000000000000000010000001000000000001015111110
500000000000000000000000000000000000000000000000000000000000000000000010000001000000000001005111112
